src/main/java/Report.java

```diff
@@ -0,0 +1,129 @@
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Report class retrieves information about students from List<Student>
+ * and is able to generate reports based on this information.
+ */
+public class Report {
+    private final List<Student> students;
+
+    public Report(List<Student> students) {
+        this.students = students;
+    }
+
+    /**
+     * generateShortReport returns short, sketchy information - one line per student.
+     */
+    public void generateShortReport() {
+
+        for (Student student : students) {
+            System.out.printf("%s(%s) - ", student.getName(), student.getCurriculum());
+
+            int programDuration = 0;
+            for (Map.Entry<String, Integer> entry : student.getCourseMap().entrySet()) {
+                programDuration += entry.getValue();
+            }
+
+            int days;
+            if (TrainingTimeManager.isTrainingFinished(student.getStartDateTime(), programDuration)) {
+                System.out.print("Training completed - ");
+                int hoursAfterCompletion = (TrainingTimeManager.calculateHoursToTrainingCompletion(
+                        student.getStartDateTime(), programDuration));
+
+                if (hoursAfterCompletion < 8) {
```

Comment:
Please avoid using "magic numbers". It would be better to have a constant like WORKING_HOURS_PER_DAY or smth like this more appropriate to the context